src/main/java/servlets/AccountServlet.java

```diff
@@ -1,17 +1,27 @@
 package com.google.sps.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/account.html")
+@WebServlet("/account")
 public class AccountServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>This is the account page.</h1>");
+    response.setContentType("text/html");
+
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      String userEmail = userService.getCurrentUser().getEmail();
+      response.getWriter().println("<p>Hello " + userEmail + "!</p>");
+    } else {
+      response.sendRedirect("/login");
```

Comment:
I think we need to change 'response.sendRedirect("/login");' to 'response.sendRedirect("/login.html");' so that the HTML displays.